src/transformState.js

```diff
@@ -5,7 +5,26 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const i of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
Завжди старайся давати назви для змінних, які щось означають